build.gradle

```diff
@@ -45,6 +45,11 @@ repositories {
     mavenCentral()
 }
 
+wrapper {
+    gradleVersion = '6.5'
+    distributionType = Wrapper.DistributionType.BIN
+}
```

Comment:
This shouldn't be necessary if we don't use Gradle 6.5 features (same thing for forcing the use of JDK 11).

Reply:
It might be necessary IMHO. 
I dunno why, but when I start a gradle project with IntelliJ IDEA, gradle's version is 6.1 by default and I don't know how to change that (and 'if' I can change that btw...). 
But Spring Boot needs an ulterior gradle version to work.  (>6.3 if I remember correctly)
Concerning jdk 11, we might use some specific features... someday.

Anyway, I'll remove that from the file, but keep the stuff in mind.